css/styles.css

```diff
@@ -0,0 +1,148 @@
+* {
+  box-sizing: border-box;
+}
+
+article {
+  background-image: url("../images/coworking.jpg");
+  background-repeat: no-repeat;
+  background-position: center top;
+  height: 400px;
+  border: 1px solid #333;
+  width: 100%;
+  float: left;
+}
+
+body {
+  margin: 0 auto;
+  padding: 54px;
+  color: black;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  background: #FFF;
+  color: #444;
+  width: 1024px;
+}
+
+header {
+  color: white;
+  background-color: black;
+  border-bottom: 6px solid yellow;
+  clear: both;
+  padding: 5px 10px 5px 10px;
+}
+
+header h2 {
+  color: yellow;
+}
+
+header a:link {
+    color: white;
+    text-decoration: none;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    margin-right: 30px;
+    margin-left: 30px;
+}
+
+.standoutLink {
+  font-weight: bold;
+ } 
+
+.subheader {
+  background-color: lightgrey;
+  color: black;
+  font-weight: bold;
+  text-align: center;
+  padding-bottom: 5px;
+  padding-top: 5px;
+  font-style: italic;
+}
+
+#crossedOutTaste {
+  color: grey;
+  text-decoration: line-through;
+}
+
+#underlinedAre {
+  text-decoration: underline;
+  text-decoration-color: red;
+  text-decoration-style: dashed;
+}
+
+section {
+  background-color: lightgrey;
+  float: left;
+  width: 30%;
+  height: 200px;
+  margin: 5px;
+  padding: 5px;
+  clear: right;
+}
+
+.sectionLeft {
+  height: 260px;
+  background-color: white;
+  float: left;
+  width: 30%;
+  margin-top: 15px;
+}
+
+.sectionRight {
+  height: 260px;
+  background-color: white;
+  float: right;
+  width: 30%;
+  margin-top: 15px;
+  margin-right: 0%
+}
+
+.sectionCenter {
+  /*float: right;*/
+  height: 260px;
+  margin-left: 4%;
+  margin-right: 4%;
+  background-color: white;
+  margin-top: 15px;
+}
+
+.floatingBox {
+  width: 30%;
+  height: 325px;
```

Comment:
You can omit the height, in the future try to let the height set the content for your components. When the content is greater that the defined height it can break your component and design.  🔨